packages/shared/src/graphql/squads.ts

```diff
@@ -187,6 +187,14 @@ export const ADD_POST_TO_SQUAD_MUTATION = gql`
   }
 `;
 
+export const UPDATE_SQUAD_POST_MUTATION = gql`
+  mutation UpdateSquadPost($id: ID!, $sourceId: ID!, $commentary: String) {
```

Comment:
Why do we need the `sourceId` as part of the parameters? Should the ID alone make it unique enough?

Reply:
not really needed, but does not hurt to have it either, my reasoning is to query based on both source & id to ensure we are not accidentally accessing the wrong record.

Reply:
I would also just stick with ID it's uniform with what we already use for other pieces and reduces overhead I think.